Example/src/LightboxExample.tsx

```diff
@@ -34,7 +35,36 @@ const NUMBER_OF_IMAGES = 4;
 const IMAGE_SIZE =
   (dimensions.width - GUTTER_WIDTH * (NUMBER_OF_IMAGES - 1)) / NUMBER_OF_IMAGES;
 
-function ImageList({ images, onItemPress }) {
+type ExampleImage = {
+  uri: string;
+  width: number;
+  height: number;
+};
+type ActiveExampleImageProperties = {
+  x: Animated.SharedValue<number>;
+  y: Animated.SharedValue<number>;
+  width: Animated.SharedValue<number>;
+  height: Animated.SharedValue<number>;
+  imageOpacity: Animated.SharedValue<number>;
+};
+type ActiveExampleImage = ActiveExampleImageProperties & {
+  // @ts-ignore: FIXME AnimatedImage type
+  animatedRef: RefObject<ActiveExampleImage>;
+  item: ExampleImage;
+};
+
+type onItemPressFn = <T>(
+  animatedRef: RefObject<T>,
+  item: ExampleImage,
+  svs: ActiveExampleImageProperties
+) => void;
+function ImageList({
```

Comment:
Inconsistency with `const Name: FC<...>` syntax used throughout examples

Reply:
Fixed in 5ba3ad1.